src/app/cube/details/components/academic-level-card/academic-level-card.component.scss

```diff
@@ -0,0 +1,40 @@
+@import '~_vars.scss';
+
+.academic-level-card {
+    border-radius: 5px;
+    box-sizing: border-box;
+    box-shadow: 0 4px 22px 0 rgba(0, 0, 0, 0.1);
+    color: $dark-grey;
+    height: 350px;
+    text-align: center;
+    width: 275px;
+    padding: 30px;
+    -moz-box-sizing: border-box;
```

Comment:
Don't think we need these, believe node-sass (the package Angular uses under the hood) automatically takes care of appending the necessary browser prefixes.